### ISSUE TITLE
Quest: 'Ledger from Tanaris' shows Complete in log, all items in inventory, but unable to turn in

### ISSUE THREAD

**Issue by cilution** [LightsHope/issues#153]
Not sure if this is a bug with the quest, or a bug with the quest items on my character.

Part of the quest has you loot 20 spider legs. I had 2 extra, and deleted those 2 to save inventory space, leaving only the required 20 in my inventory. I suspect that may have something to do with not being able to complete the quest. I've attempted logging out and in since doing that to no avail.

![image](https://user-images.githubusercontent.com/662945/37795804-82d7cb5c-2de3-11e8-8254-b615547e4ce3.png)
![image](https://user-images.githubusercontent.com/662945/37795820-908cba50-2de3-11e8-87c9-b1b0c418baf9.png)

**Comment by Starfallen1** [comment]
iirc (its been awhile since I did any leveling) sometimes putting items in your bank will bug things out, try dropping the quest and picking it back up.

**Comment by cilution** [comment]
No banking was involved here, so that can't be the issue.

I doubt I'll get around to testing dropping it and picking it up again, you get it out in the middle of no where in Searing Gorge and turn it in in Tanaris, far too inconvenient.

**Comment by Niceguyone** [comment]
This quest has been bugged in the same way on all the private servers I have played on and I have never managed to return it on first attempt. Instead, I have always had to go back to Searing Gorge to retake it. 

Also for some people putting your quest items in and out of bank has worked (just like Starfallen said).

**Comment by druidish** [comment]
Did you delete any extra [Solid Crystal Leg Shaft] that can also cause this issue.

**Comment by Ripper007** [comment]
yes, never delete the extra leg shafts and the quest can be delivered. The other workaround is to abandon the quest and retake it.

**Comment by crackyoo** [comment]
Hey, this happens for all quests when you get more than the quest items you need and delete the extra ones. Also for quests such as Soothing Turtle Bisque where the item needed isn't a quest item.

**Comment by GuybrushGit** [comment]
Can't reproduce, tried deleting, banking etc nothing causes this in dev.

**Comment by bergner** [comment]
I can confirm I have exactly the same problem on Darrowshire just now. Have all 4 items but Krinkle doesn't allow me to complete the quest. Tried putting items in bank, logging out etc. but only my only choice seems to be to go back to Searing Gorge and pick up the quest again. I know the same problem existed months ago when I leveled other characters.

**Comment by Polopollo85** [comment]
Had the issue here: https://github.com/LightsHope/issues/issues/302#issuecomment-387731060

For science this morning, I dropped the quest, went to Kadgart getting it again, then heartstoned to Orgri, then flew to Tanaris. And i was finally able to turn it in.
